Subsurface/FlightComputer/include/NetClient.h

```diff
@@ -0,0 +1,236 @@
+#pragma once
+#define ASIO_STANDALONE
+#include <asio.hpp>
+
+#include "Message.h"
+#include "Paradigm.pb.h"
+
+#include <deque>
+#include <iostream>
+
+using asio::ip::tcp;
+using asio::ip::udp;
+
+template<typename Proto>
```

Comment:
Probably look at making this more general/not templated so we can send multiple protobuf message types over one tcp client

Reply:
So the reason this is templated is because the Message class is templated. Thus if we wanna make this more general to work with multiple types of Protobuf messages then we need to figure out a way to make the Message class not templated. Possible solution:
 - Remove any references to Protobuf template type in Message class (i.e. remove serialize and deserialize methods, and make the Message class just a fixed header, and then a body of variable length). Then if we want to add functionality specific to certain types of Protobuf messages then we just inherit from the Message class, and that will remove templating from the picture.

Reply:
I am gonna add some commits to do this and then we can decide how we wanna proceed.